profiling-crashtracking-receiver/src/main.rs

```diff
@@ -36,6 +39,9 @@ pub fn main() -> anyhow::Result<()> {
                 // TODO Experiment to see if 30 is the right number.
                 crash_info.upload_to_dd(endpoint, Duration::from_secs(30))?;
             }
+            if let Some(uploader) = telemetry_uploader {
+                uploader.upload_to_telemetry(&crash_info, Duration::from_secs(30))?;
```

Comment:
Should we make 30 seconds configurable? Or is that a standard value?

Reply:
I'm not sure, there needs to be a timeout, but 30s seems too much for me since we're blocking the parent process until we return, but I think the evp intake the data goes too after being proxied by the agent out can have pretty long outliers from time to time (requests blocking for up to 15s in rare cases).

But I don't know if it's a good idea to let users modify it 🤔 Maybe the crashtracker should daemonize itself and do crash submission asynchronously after we receive it, and then we can keep the long timeout.

Reply:
The issue which I'm worried about is the pod being killed when the parent process dies, and the crash-tracker info never getting out.

Reply:
Hmm true, and if the crash tracker daemonize itself if the crashing process is not PID 1 in the container we might end up with zombie processes

Reply:
Also, if we block the process from exiting for a long time we might prevent any orchestrator from restarting it and worsen the impact a crash might have on availability